squid/templates/configmap.yaml

```diff
@@ -7,12 +7,122 @@ metadata:
     {{- include "squid.labels" . | nindent 4 }}
 data:
   squid.conf: |-
-    {{- .Files.Get "squid.conf" | nindent 4 }}
+    # ACL definitions - RFC 1918 private networks, link-local, and shared address space
+    acl localnet src 0.0.0.1-0.255.255.255 10.0.0.0/8 100.64.0.0/10 169.254.0.0/16
+    acl localnet src 172.16.0.0/12 192.168.0.0/16 fc00::/7 fe80::/10
+
+    # ACL definitions - SSL and safe ports
+    acl SSL_ports port 443 8443
+    acl Safe_ports port 80 21 443 70 210 1025-65535 280 488 591 777
```

Comment:
I assume AI may have clobbered these lines for w/e reason. Most people won't know what many of these sources or ports are used for off the top of their head. It was helpful to have comments here. Please restore them.